blockchain/blockchain_test.go

```diff
@@ -724,3 +724,55 @@ func TestSubscribeL1Head(t *testing.T) {
 	require.True(t, ok)
 	assert.Equal(t, l1Head, got)
 }
+
+func fetchStateUpdatesAndBlocks(samples int) ([]*core.StateUpdate, []*core.Block, error) {
+	client := feeder.NewClient(utils.Mainnet.FeederURL)
+	gw := adaptfeeder.New(client)
+
+	suList := make([]*core.StateUpdate, samples)
+	blocks := make([]*core.Block, samples)
+	for i := range samples {
+		fmt.Println("fetching", i)
+		su, err := gw.StateUpdate(context.Background(), uint64(i))
+		if err != nil {
+			return nil, nil, err
+		}
+		suList[i] = su
+		block, err := gw.BlockByNumber(context.Background(), uint64(i))
+		if err != nil {
+			return nil, nil, err
+		}
+		blocks[i] = block
+	}
+	return suList, blocks, nil
+}
+
+func BenchmarkBlockchainStore(b *testing.B) {
```

Comment:
We should write this in a separate file, because this file is already quite long.

Reply:
actually removed this benchmark, its not used anymore and not very precise